public/app/core/utils/richHistory.ts

```diff
@@ -330,7 +330,7 @@ export function filterQueriesBySearchFilter(queries: RichHistoryQuery[], searchF
     const listOfMatchingQueries = query.queries.filter(query =>
       // Remove fields in which we don't want to be searching
       Object.values(_.omit(query, ['datasource', 'key', 'refId', 'hide', 'queryType'])).some((value: any) =>
```

Comment:
NitPick: would `filter` (filtering out null|undefined) before `some` be a more readable solution?

Reply:
Good question. Honestly, I am not sure. For me personally, both are equally readable. Also would it make sense to run 2 array methods instead of just 1?

Reply:
Just a suggestion, no biggie at all.